src/app/app.module.ts

```diff
@@ -10,6 +10,7 @@ import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app-routing.module';
 import {HttpClientModule} from "@angular/common/http";
```

Comment:
tiny consistency thing - super not noticeable really - but added the space for line 11 here for HttpClientModule - I'll push it to the branch.